roles/backup/tasks/main.yml

```diff
@@ -1,16 +1,28 @@
 ---
 
-- name: set up weekly backup
+- name: install backup script
   become: yes
   template:
-    src: etc/cron.weekly/cnx-backup
-    dest: /etc/cron.weekly/cnx-backup
+    src: usr/local/bin/cnx-backup
+    dest: /usr/local/bin/cnx-backup
     mode: 0755
 
-- name: set up slim dumps
+- name: set up weekly backup
+  become: yes
+  src: /usr/local/bin/cnx-backup
+  dest: /etc/cron.weekly/cnx-backup
+  state: link
+
+- name: install slim dump script
   become: yes
   when: slim_dumps|default(False)
   template:
-    src: etc/cron.weekly/cnx-slim-dump
-    dest: /etc/cron.weekly/cnx-slim-dump
+    src: /usr/local/bin/cnx-slim-dump
+    dest: /usr/local/bin/cnx-slim-dump
     mode: 0755
+
+- name: set up slim dumps
+  become: yes
```

Comment:
This task is missing `when: slim_dumps|default(False)`, it creates a broken link if `slim_dumps` is no.